config/custom-environment-variables.json

```diff
@@ -1,21 +1,19 @@
 {
   "passport": {
     "Google": {
-      "clientID": "GOOGLE_CLIENT_ID",
-      "clientSecret": "GOOGLE_CLIENT_SECRET",
-      "callbackURL": "GOOGLE_CALLBACK_URL"
+      "clientID": "315399887741-3j1g3nfpm1bb22gho31tadb2ug4h2nlo.apps.googleusercontent.com",
```

Comment:
😵

Reply:
I don't know why this commit always gets here but these are the old client secrets so all good